scripts/clean_containers.py

```
@@ -0,0 +1,12 @@
+from constants import *
+import subprocess
+
+def clearContainers():
+    cmd = ["docker", "rm", DLV_CONTAINER_NAME]
```

Ruff (0.8.2)

`DLV_CONTAINER_NAME` may be undefined, or defined from star imports

(F405)